File: api/app/Models/Posto.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Posto extends Model {

    use HasFactory;

    protected $fillable = ['cidade_id', 'reservatorio', 'latitude', 'longitude'];

    public function cidade() {
        return $this->belongsTo(Cidade::class);
    }

}

File: models/posto.py
from datetime import datetime
from sqlite3 import Timestamp
from typing import Optional
import ormar
from config import database, metadata

from models.cidade import Cidade

class Posto(ormar.Model):
  class Meta:
    metadata = metadata
    database = database
    tablename = 'postos'

  id: int = ormar.Integer(primary_key=True)
  reservatorio: int = ormar.Integer() # vai ser representado em porcentagem
  latitude: float = ormar.Float()
  longitude: float = ormar.Float()
  # created_at: Timestamp = ormar.DateTime(timezone=True)
  # updated_at: Timestamp = ormar.DateTime(timezone=True)
  cidade_id: int = ormar.ForeignKey(Cidade) #FOREIGN KEY